client/package.json

```diff
@@ -24,7 +25,7 @@
 	},
 	"dependencies": {
 		"semver": "^5.5.0",
-		"vscode-languageserver-protocol": "3.14.1"
+		"vscode-languageserver-protocol": "file://../protocol"
```

Comment:
This shouldn't be necessary. After cloning `vscode-languageserver-node` and running npm install simply run `npm run symlink` which does a developement setup linking.